packages/core/src/browser/quick-input/quick-command-frontend-contribution.ts

```diff
@@ -38,6 +38,9 @@ export class QuickCommandFrontendContribution implements CommandContribution, Ke
         commands.registerCommand(CLEAR_COMMAND_HISTORY, {
             execute: () => commands.clearCommandHistory(),
         });
+        commands.registerCommand({ id: 'workbench.action.closeQuickOpen' }, {
```

Comment:
minor - refactor the command to a constant (since it is referenced multiple times)?

```ts
export const CLOSE_QUICK_OPEN: Command = {
    id: 'workbench.action.closeQuickOpen'
}
```